bender-xilinx.mk

```diff
@@ -21,6 +21,11 @@ $(eval $(call check_enable_island,GEN_PULP_CLUSTER))
 $(eval $(call check_enable_island,GEN_SAFETY_ISLAND))
```

Comment:
I recommend reading the documentation first to understand the new ext_jtag parameter